front-end/src/components/newsfeed-view/news-list/NewsList.js

```diff
@@ -3,12 +3,13 @@ import  _  from 'lodash';
 import { formatDate } from 'utils/utils';
 import pic from 'assets/images/cutiepie.jpg';
 import './newslist.css';
-import NewsCardGroup from './NewsCardGroup';
+import NewsCardGroup from './card-group/NewsCardGroup';
 
 const dummyfeed = [
```

Comment:
extra this data to `__mock__` directory, if exist. Because you would need this for unit testing and itegration testing anyway.
 (mocking redux behaviors, mocking network api calls, etc)